koku/masu/database/sql/openshift/all/reporting_ocpallcostlineitem_project_daily_summary_gcp.sql

```diff
@@ -58,10 +57,6 @@ SELECT 'GCP' as source_type,
        max(unit) as unit,
        sum(unblended_cost + credit_amount) as unblended_cost,
        sum(project_markup_cost) as project_markup_cost,
```

Comment:
```suggestion
       sum(markup_cost) as project_markup_cost,
```
Did we miss this? Your removing this column here https://github.com/project-koku/koku/pull/5582/files#diff-57da9ba93f33030c8a1b0702f3856d401414101674d4147a3099502de56189cdL154